app/scripts/i18n/react/locales/en/translations.json

```
@@ -33,5 +33,34 @@
             "failed_to_scan_error": "Scan failed for ports: {{ failed_ports }}. Check logs for more information"
          }
       }
+   },
+   "network-connections": {
+      "title": "Network connections",
+      "labels": {
+         "changes": "There are unsaved changes",
+         "has-errors": "There are invalid parameters",
+         "activated": "Connected",
+         "activating": "Connecting",
+         "deactivating": "Disconnecting",
+         "not-connected": "Not connected",
+         "deprecated": "Deprecated",
+         "new": "New connection",
+         "select-type": "Select connection type",
+         "wifi": "Wi-Fi",
+         "wifi-ap": "Wi-Fi access point",
+         "ethernet": "Ethernet",
+         "modem": "GSM-modem",
+         "canbus": "CANBUS",
```

```suggestion
         "canbus": "CAN bus",
```

И в русской локали так же, кажется, аккуратней

---

Поменял